arch/arm/mach-bcm283x/init.c

```diff
@@ -165,6 +165,27 @@ static void rpi_update_mem_map(void)
 static void rpi_update_mem_map(void) {}
 #endif
 
+#ifdef CONFIG_BCM2712
+static void rpi_set_pclk_rate(void)
+{
+	int len;
+	const int *cell;
+	int offset = fdt_path_offset(gd->fdt_blob, "/clocks/macb_pclk");
+
+	if (!offset)
```

Comment:
It would be nice to print some error message

Reply:
same as above. removed